code/modules/mob/living/carbon/human/life.dm

```diff
@@ -892,9 +892,9 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 
 		//Vision //god knows why this is here
-		var/obj/item/organ/vision
+		var/obj/item/organ/internal/vision
 		if(species.vision_organ)
-			vision = internal_organs_by_name[species.vision_organ]
+			vision = get_int_organ(/obj/item/organ/internal/eyes)//wot?

```

Comment:
It's so IPC eyes can exist under a separate organ_tag, so that they don't have "eyes" when welding and thus harming themselves, and also so that Diona and Slime People aren't blind forever.